app.js

```diff
@@ -0,0 +1,123 @@
+const express = require('express');
+const morgan = require('morgan');
+
+const app = express();
+
+app.use(express.json());
+app.use(morgan('dev'));
+
+app.get('/users', async (req, res) => {
+    try {
+        const pageNumber = req.query.page;
+        const response = await fetch(`https://reqres.in/api/users?page=${pageNumber}`);
+        const users = await response.json();
+
+        if (!users) {
+            res.status(404).json({ message: 'Users Not found' });
```

Comment:
create a new file called `errors.js` it will have some functions like `getNotFoundResponse(res)` and `getUnknownErrorResponse(res) `